crates/hyperdrive-wrappers/build.rs

```diff
@@ -1,17 +1,19 @@
 use std::{
     collections::HashMap,
     env,
-    fs::{create_dir_all, read_to_string, OpenOptions},
+    fs::{self, create_dir_all, read_to_string, OpenOptions},
```

Comment:
are you sure you want to import fs::self here? that seems odd

Reply:
is it because of `fs::read_dir`? could you just import `read_dir` and not self and then avoid the `fs::` later?

Reply:
yeah good call, fixed this.